docs/sources/developer/Dependencies.md

```diff
@@ -1,48 +1,36 @@
 # Plaso Dependencies
 
-This page contains detailed instructions on how to build and install dependencies.
+This page contains detailed instructions on how to install dependencies for 
+development.
 
 There are multiple ways to install the dependencies:
 
-* Using [prepackaged dependencies](Dependencies.md#prepackaged-dependencies);
-* [Batch build](Dependencies.md#batch-build) most of the dependencies;
-* [Manual build](Dependencies.md#manual-build) of the dependencies.
+* [Using pre-packaged dependencies](Dependencies.md#prepackaged-dependencies)
```

Comment:
both writing forms are valid, I opt to stick with 1 since that was prepackaged please revert this change